Ginger/GingerCoreNET/Database/NoSqlBase/GingerHbase.cs

```diff
@@ -0,0 +1,611 @@
+                resarray[0] = "NotEqual";
+            }
+            else if (refstring.Contains("<="))
+            {
+                words1 = refstring.Split("<=");
+                resarray[0] = "LessThanOrEqualTo";
+            }
+            else if (refstring.Contains(">="))
+            {
+                words1 = refstring.Split(">=");
+                resarray[0] = "GreaterThanOrEqualTo";
+            }
+
+            else if (refstring.Contains('='))
+            {
+                words1 = refstring.Split('=');
+                resarray[0] = "Equal";
+            }
+            else if (refstring.Contains('<'))
+            {
+                words1 = refstring.Split('<');
+                resarray[0] = "LessThan";
+            } 
+
+            else if (refstring.Contains('>'))
+            {
+                words1 = refstring.Split('>');
+                resarray[0] = "GreaterThan";
+
+            }
+           
+
+            if (words1[0].Contains(':'))
+            {
+                string[] temp = words1[0].Split(':');
+                resarray[1] = Base64Encode(temp[0].Trim()); // Family name
+                resarray[2] = Base64Encode(temp[1].Trim()); // Field name
+                resarray[3] = Base64Encode(words1[1].Trim()); // Field value
+            }
+            else
+            {
+                resarray[1] = "";    //Family Name
+                resarray[2] = words1[0].Trim();  // Field Name
+                resarray[3] = words1[1].Trim();  // Field Value
+
+            }
+            return resarray;
+
+        }
+        public Filter getFilter(string op, string family, string fieldName, string fieldValue)
+        {
+            CompareFilter.CompareOp compareOp = Enum.GetValues<CompareFilter.CompareOp>().FirstOrDefault(e => string.Equals(e.ToString(), op));
+            if (string.Equals(compareOp.ToString(), op))
+            {
+                return new SingleColumnValueFilter(Encoding.UTF8.GetBytes(family), Encoding.UTF8.GetBytes(fieldName), compareOp, Encoding.UTF8.GetBytes(fieldValue));
+            }
+            else if (string.Equals("RegexComp", op))
+            {
+                RegexStringComparator comp = new RegexStringComparator(fieldValue);   // any value that starts with 'my'
+                return new SingleColumnValueFilter(
+                  Encoding.UTF8.GetBytes(family),
+                  Encoding.UTF8.GetBytes(fieldName),
+                  CompareFilter.CompareOp.Equal,
+                  comp);
+            }          
+            else
+            {
+                //no CompareOp enum value found matching the given op
+                return null;
+            }
+            
+        }
+
+        public override async void PerformDBAction()
+        {
+            try
+            {
+                
+                ValueExpression VE = new ValueExpression(Db.ProjEnvironment, Db.BusinessFlow, Db.DSList);
+                VE.Value = Act.SQL;
```

Comment:
Please use QueryValue instead of SQL here. Do merge from Beta branch if this property is not available